### ISSUE TITLE
navigationOptions.headerStyle doesn't support style array

### ISSUE THREAD

**Issue by dgruseck** [react-navigation/react-navigation#3608]
### Current Behavior

- You can not pass an array of styles to headerStyle in the navigationOptions.
This was introduced with the commit #3568.

### Expected Behavior

- It should be possible to pass an array of styles or a single style object.

### How to reproduce

```javascript
navigationOptions: () => ({
  headerStyle: [styles.headerStyle, styles.bg_red],
}),
```

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 1.2.0
| react-native     | 0.53.3

**Comment by brentvatne** [comment]
hm ok. thanks for pointing this out, will investigate, sorry for hassle

**Comment by brentvatne** [comment]
fixed in 1.2.1! thank you
